hr_attendance_report_theoretical_time/models/resource_calendar.py

```diff
@@ -0,0 +1,18 @@
+# Copyright <2019> Pesol <pedro.gonzalez@pesol.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class ResourceCalendar(models.Model):
+    _inherit = "resource.calendar"
+    _description = "Resource Working Time"
+
+    def _work_intervals(self, start_dt, end_dt, resource=None, domain=None):
```

Comment:
Why is this needed?

Reply:
if function _attendance_intervals does not receive a resource (in this case the employee) then a search on object "resource.calendar.attendance" with domain (resource_in in [False]) will be triggered with no result found. I'm opened to other solutions.

Reply:
as @ernestotejeda pointed in his comment using the private method _get_work_days_data fix this issue, so no need to rewrite the function _work_intervals